src/lib/loadingscreen/index.css

```diff
@@ -0,0 +1,32 @@
+h1 {
+	font-family: mojangles, minecraft, monospace;
+}
+.loader {
+	display: initial;
```

Comment:
As discussed, I'd prefer not separating css and html and js in different files
Is there a benefit?